code/Weapons/Explosives/Components/ExplosiveComponent.cs

```diff
@@ -0,0 +1,59 @@
+namespace Grubs;
+
+public partial class ExplosiveComponent : EntityComponent<Explosive>
+{
+	protected Explosive Explosive => Entity;
+	protected Grub Grub => Explosive.Grub;
+	protected Player Player => Grub.Player;
+
+	public virtual void Simulate( IClient client )
+	{
+		PhysicsTick();
```

Comment:
This might be a bad idea to do this here... every sngle explosive component will be constantly calling this.